app/components/av-table.js

```diff
@@ -6,6 +6,8 @@ export default Ember.Component.extend({
 
   didReceiveAttrs() {
     this.set('tableData', this.get('data'));
+
+    // must come from caller
     this.set('skip', 0);
```

Comment:
I feel it should be encapsulated. Unless there is a requirement to bookmark list pages.